server/src/utils/jwt.ts

```diff
@@ -0,0 +1,21 @@
+import { SignOptions, sign, verify } from 'jsonwebtoken';
+
+export function signJwt<T extends string | object | Buffer>(
+  object: T,
+  secret: string,
+  options?: SignOptions,
+) {
+  const signingKey = Buffer.from(secret, 'base64').toString('ascii');
+
+  return sign(object, signingKey, { ...options, algorithm: 'HS256' });
+}
+
+export function verifyJwt<T>(token: string, secret: string): T | null {
+  const publicKey = Buffer.from(secret, 'base64').toString('ascii');
+
+  try {
+    return verify(token, publicKey, { complete: false }) as T;
```

Comment:
the `HS256` algorithm is symmetric, signing and verification process share exact same passphase, therefore the variable name "publicKey" at line 17 is misleading.

Reply:
Yes, good point.   These functions are actually designed for all kinds of JWT algorithms.  I have rename the variable to `verifyKey` in https://github.com/Wiredcraft/test-fullstack/pull/78/commits/c3e61ecc9fea5942bc5c197184294e201398450d